staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/vault/client.go

```diff
@@ -0,0 +1,203 @@
+package vault
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/vault/api"
+)
+
+// Handle all communication with Vault server.
+type clientWrapper struct {
+	client      *api.Client
+	encryptPath string
+	decryptPath string
+        authPath    string
```

Comment:
A `go fmt` would be good as there's some mixed/misaligned indentation here and a couple of other places.